### ISSUE TITLE
Gene in many orthogroups

### ISSUE THREAD

**Issue by EmilyPhelps** [davidemms/OrthoFinder#944]
Hello, 

I am using orthofinder to connect genes to do some cross species transcriptomic analysis.
When I dig into the results, some genes (from a single species/transcriptome) are represented in multiple orthogroups.

Is this biologically possible? Is this an artifact of using orthogroups on transcriptome data?

I have used only the default parameters but have run my transcriptomes through CD-hit and kept only the sequences longer than 50 amino acids. I also ran Transdecoder and am only using the longest isoform.

Any advice is much appreciated

**Comment by lauriebelch** [comment]
Hi Emily,

This shouldn't happen, so might be an artifact of using transcriptome data

If you send me an output file (Phylogenetic_Hierarchical_Orthogroups/N0.tsv), an input file, and the name of a gene that appears in multiple orthogroups I can take a look

**Comment by EmilyPhelps** [comment]
Hi Laurie,

I managed to solve this issue. In summary: It was an isoform issue not a Orthofinder issue!

Incase anyone else has a similar problems: I was using the transdecoder run from the Trinity genome assembly process but this retained multiple isoforms. To solve this I filtered my final transcriptome by the longest isoform and then put this into Orthofinder. Which gives one orthogroup per gene.
